react-apps/self-care-portal/self-care-portal-frontend/accelerator/src/api/consent-api.js

```diff
@@ -170,3 +172,45 @@ export const getConsentsFromAPIForSearch = (searchObj, user, appInfo) => {
         });
 };
 
+export const withdrawConsent = async (
```

Comment:
Shall we rename this to revokeConsent?

Reply:
resolved in [106de5c](https://github.com/wso2/financial-open-banking/pull/49/commits/106de5ca75429194a33fa5474cba8b5fbdc115ff)